packages/block-library/src/block/edit.native.js

```diff
@@ -0,0 +1,262 @@
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { BlockEditorProvider, BlockList } from '@wordpress/block-editor';
+import { compose, withPreferredColorScheme } from '@wordpress/compose';
+import { parse } from '@wordpress/blocks';
+import { help } from '@wordpress/icons';
+import { requestUnsupportedBlockFallback } from '@wordpress/react-native-bridge';
+
+/**
+ * Internal dependencies
+ */
+import styles from './editor.scss';
+import EditTitle from './edit-title';
+
+class ReusableBlockEdit extends Component {
+	constructor( { reusableBlock } ) {
+		super( ...arguments );
+
+		this.setBlocks = this.setBlocks.bind( this );
+		this.toggleSheet = this.toggleSheet.bind( this );
+		this.requestFallback = this.requestFallback.bind( this );
+
+		if ( reusableBlock ) {
+			// Start in edit mode when we're working with a newly created reusable block
+			this.state = {
+				// Since edition is not supported yet isEditing is always false
+				isEditing: false,
+				blocks: parse( reusableBlock.content ),
+				showHelp: false,
+				sendFallbackMessage: false,
+			};
+		} else {
+			// Start in preview mode when we're working with an existing reusable block
+			this.state = {
+				isEditing: false,
+				blocks: [],
+				showHelp: false,
+				sendFallbackMessage: false,
+			};
+		}
+	}
+
+	componentDidMount() {
+		this.props.fetchReusableBlock();
+	}
+
+	componentDidUpdate( prevProps ) {
+		const hasBlockFetchCompleted =
+			! prevProps.reusableBlock && this.props.reusableBlock;
+		const hasBlockContentChanged =
+			prevProps.reusableBlock?.content !==
+			this.props.reusableBlock?.content;
+
+		if ( hasBlockFetchCompleted || hasBlockContentChanged ) {
+			this.setState( {
+				blocks: parse( this.props.reusableBlock.content ),
+			} );
+		}
+	}
+
+	toggleSheet() {
+		this.setState( {
+			showHelp: ! this.state.showHelp,
+		} );
+	}
+
+	requestFallback() {
+		this.toggleSheet();
+		this.setState( { sendFallbackMessage: true } );
+	}
+
+	setBlocks( blocks ) {
+		this.setState( { blocks } );
+	}
+
+	renderSheet() {
+		const { reusableBlock } = this.props;
+		const { showHelp } = this.state;
+
+		const { getStylesFromColorScheme, clientId } = this.props;
+		const infoTextStyle = getStylesFromColorScheme(
+			styles.infoText,
+			styles.infoTextDark
+		);
+		const infoTitleStyle = getStylesFromColorScheme(
+			styles.infoTitle,
+			styles.infoTitleDark
+		);
+		const infoSheetIconStyle = getStylesFromColorScheme(
+			styles.infoSheetIcon,
+			styles.infoSheetIconDark
+		);
+		const actionButtonStyle = getStylesFromColorScheme(
+			styles.actionButton,
+			styles.actionButtonDark
+		);
+
+		const infoTitle =
+			Platform.OS === 'android'
+				? __(
+						"Reusable blocks aren't editable on WordPress for Android"
+				  )
+				: __( "Reusable blocks aren't editable on WordPress for iOS" );
+
+		return (
+			<BottomSheet
+				isVisible={ showHelp }
+				hideHeader
+				onClose={ this.toggleSheet }
```

Comment:
Oh ok I'll create a close function for this case then.
I used part of the logic from the `missing` block to do this because the intended behavior was almost the same, so I'll review the `missing` block too because this issue could be happening there.

Reply:
Here is the [commit](https://github.com/WordPress/gutenberg/pull/25265/commits/f99ef14d255a2234579140cea225d5c8ee2421e1) that fixes this issue.

I also checked the issue I commented above related to the `missing` block and it's also happening, on Android if you rotate the device while you have a `missing` block on a post, the bottom sheet is presented. The fix is quite easy so it could be done in this PR.

Reply:
> I also checked the issue I commented above related to the missing block and it's also happening, on Android if you rotate the device while you have a missing block on a post, the bottom sheet is presented. The fix is quite easy so it could be done in this PR.

Thanks for checking that! There seems to be no existing issue for this so I created one [here](https://github.com/wordpress-mobile/WordPress-Android/issues/13052).